### DISCUSSION TITLE
Hits and connectHits not working even when InstantSearch has successfully returned the result.

### DISCUSSION THREAD

**Discussion by dennism501** [algolia/instantsearch#5292] [Q&A] [answered]
I am using a algolia index to get a certain dataset, the request is successful but when I try to access the hits using `Hits` or the `connectHits` components. The props are empty, I don't not get it. HELP!!

**Comment by Haroenv** [comment, marked as answer]
Can you reproduce in a codesandbox? We have an example available here http://codesandbox.io/s/github/algolia/create-instantsearch-app/tree/templates/react-instantsearch

**Reply by dennism501** [reply]
Yes, let me reproduce it. Just a moment

**Reply by dennism501** [reply]
I managed to fix it, I just had to rewrite my code :) Thanks.

**Reply by Haroenv** [reply]
glad to hear you fixed it! What was the issue in the end?

**Reply by dennism501** [reply]
So the issue was configuration on the `<Configure />` I was applying the setting wrongly and it was 
```
Wrong config
<Configure facets={"*"} facetFilters="[["planet_code:EAR"]]" />

Correct Config
<Configure facets{["*"]} facetFilters={[["planet_code:EAR"]]} />
```

This seemed to work, could you say if I am using the Configure component correctly? I really gave me hell the past few days.

**Reply by Haroenv** [reply]
it seems correct, but it depends on what you are trying to achieve. In general I'd recommend using RefinementList or our other components (possible with rendering null / virtual widget). The api is much simpler than configure

**Reply by dennism501** [reply]
I just want to filter the results from the Instant search, so that's why I am using Configure and it's a requirement for my case. Now I am trying to create an autoComplete for React with my InstantSearch instance. Could you be able to help with an example of an AutoComplete using InstantSearch, the documentation is helping but not in a way i want it to.

**Reply by Haroenv** [reply]
does this example help? https://codesandbox.io/s/k9p6y7zllr

**Reply by dennism501** [reply]
It does help, I actually found this example in my search. But I am looking for is how to display suggestions that come from the first hits from the InstanctSearch before the user can start typing. Like this for instance 

![image](https://user-images.githubusercontent.com/15628660/150787543-07b29e9e-f81c-4cb9-90a6-50640a5dc994.png)

Also for the connetAutoComplete UI library, is there a way of making the text box look like the image above? I followed the example from this section of the algolia docs https://www.algolia.com/doc/ui-libraries/autocomplete/guides/using-react/ 

The output is: 

![image](https://user-images.githubusercontent.com/15628660/150787950-372a53ed-e18f-45d3-80c1-f17ddb485678.png)

**Reply by Haroenv** [reply]
you were missing the css, as mentioned in the getting started https://www.algolia.com/doc/ui-libraries/autocomplete/introduction/getting-started/#install-the-autocomplete-classic-theme

**Reply by dennism501** [reply]
Thank you very much, I do not know how I skipped this. Thanks again, another question :) , does GeoSearch have API for MapBox like it does for Google maps or do I just use `connectGeoSearch` and make my own Mapbox component?

**Reply by Haroenv** [reply]
we don't have official support for mapbox (yet?), but there's an example in a guide here: https://www.algolia.com/blog/engineering/building-a-store-locator-in-react-using-algolia-mapbox-and-twilio-part-1/

**Reply by dennism501** [reply]
I found this guide as well, I am one hour into the 3 video series, two more to go. It's the only reference I have of using Mapbox, I will continue using it. If I have some things to contribute with my findings, I will document them. Thank you for the help!
